exporter/common.go

```diff
@@ -1,3 +1,19 @@
+// mongodb_exporter
+// Copyright (C) 2017 Percona LLC
```

Comment:
2017 ? is that correct @atymchuk ?

Reply:
Actually, most of our client-related code is licensed under Apache-2.0, unless it's a fork and the upstream repo has a different license, which we need to stick to. Otherwise, let's make it Apache-2.0.

Reply:
yeah, I am not and expert, just asking why 2017 if we have 2022 :)

Reply:
Oh, I was answering the wrong question )
It doesn't make too much of a difference from the legal standpoint, but it's more logical to put 2022 :)

Reply:
You can take it from [here](https://github.com/percona/pmm-agent/blob/main/actions/actions.go#L1-L14) except for the year, which should be 2022.

Reply:
yes, that is my point :) if we adding new one it should have recent year

Reply:
> please change the year to 2022

interestingly, it checks that the license text in the target file is the same as the one in `check-license.go` (currently it's 2017 which is why CI failed) - We can keep all of them at 2017, update them all to 2022, or make `check-license.go` independent of the year.

Reply:
for me it should generate current year and check for that, and fail if there are some old headers. We probably need this check license centralized somewhere or in form of GH package to use in the all projects to not update it everywhere.

Reply:
@atymchuk what do you think ?

Reply:
I think if we want to centralize this, we'd have to add support for different licenses (since not all projects use the same license). 

Also, maybe we can keep the old headers as they are without failing them and only use newer copyright years in new files? I updated check-license to accept all years from the oldest file to $CURRENT_YEAR so you could help check if it makes sense